### ISSUE TITLE
how to click on dropdown?

### ISSUE THREAD

**Issue by joech4n** [1995eaton/chromium-vim#504]
Is there is a way to click on a dropdown menu with cvim?

**Comment by Dessix** [comment]
This may actually be two bugs in one- Tabbing to an input such that it's focused drops you into insert mode, but doing so on a dropdown doesn't activate insert mode- often I find myself typing the first few letters of a state in forms with that field, only to find that I'm now on a different page by the assumption that it'd behave as the other inputs do.

**Comment by ShawnConn** [comment]
This worked at some point. I can't remember at what point this regression started. Anyway, I've found changing [this line](https://github.com/1995eaton/chromium-vim/blob/master/content_scripts/hints.js#L194) from:

`window.setTimeout(function() { DOM.mouseEvent('click', link); }, 0);`

to:

`link.focus();`

Sets the focus on the selectbox where you can select options from it.

**Comment by 1995eaton** [comment]
I believe this feature stopped working after a Chrome update a while ago. @ShawnConn I've implemented your change, so at least you can now select the dropdown items after typing on the keyboard. I'll keep this issue open until me or someone else can find a workaround to get select boxes open via JavaScript.

**Comment by Dessix** [comment]
Space selects the focused item if you aren't using it as your leader key.

**Comment by 1995eaton** [comment]
@Dessix yep. I may add additional patch that makes cVim treat focused select boxes as input elements, so as long as they're focused, using space to open the options won't conflict with keybindings. How do you all feel about this change?

**Comment by Dessix** [comment]
@1995eaton glad of it, and it's great to see you back in development of this 🗡
